File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

var (
	reader = bufio.NewReader(os.Stdin)
	writer = bufio.NewWriter(os.Stdout)
)

func main() {
	defer writer.Flush()
	var n, m int
	var url, password string
	fmt.Fscanf(reader, "%d %d\n", &n, &m)
	var urlPw = map[string]string{}
	for i := 0; i < n; i++ {
		fmt.Fscanf(reader, "%s %s\n", &url, &password)
		urlPw[url] = password
	}
	for i := 0; i < m; i++ {
		fmt.Fscanf(reader, "%s\n", &url)
		fmt.Fprintf(writer, "%s\n", urlPw[url])
	}
}

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.py
import sys
input = sys.stdin.readline 
sys.setrecursionlimit(10**6)
n, m = map(int, input().split())
urlPw = dict()
for i in range(n):
    url, pw = map(str, input().split())
    urlPw[url]=pw
for i in range(m):
    print(urlPw[input().rstrip('\n')])